smart-contract/assembly/__tests__/FileBuilder.ts

```diff
@@ -136,16 +155,29 @@ class FileBuilder {
         `File ${fileInfo.fileName} should be in the file list`,
       );
       for (let j = 0; j < fileInfo.data.length; j++) {
-        const storedChunk = getChunk(
-          new ChunkGet(fileInfo.fileNameHash, j).serialize(),
-        );
+        const storedChunk = getChunk(chunkGetArgs(fileInfo.fileNameHash, j));
         assert(
           storedChunk.length == fileInfo.data[j].length,
           `Chunk ${j} of ${fileInfo.fileName} should have correct length`,
         );
       }
     }
   }
+
+  hasNoFiles(): void {
+    const fileList = new Args(getFilePathList()).next<string[]>().unwrap();
+    assert(fileList.length === 0, 'FileList should be empty');
```

Comment:
We can also check that there is no chunks associated with this file. and that totalchunk has been removed